pkgs/applications/editors/xedit/default.nix

```diff
@@ -22,6 +22,12 @@ stdenv.mkDerivation rec {
     sha256 = "0b5ic13aasv6zh20v2k7zyxsqbnsxfq5rs3w8nwzl1gklmgrjxa3";
   };
 
+  postPatch = lib.optionalString stdenv.isDarwin ''
```

Comment:
Please add a comment about why this is needed and if there's a upstream issue for it